### ISSUE TITLE
Show more Bookmarks on New Tab

### ISSUE THREAD

**Issue by bryanbell** [mozilla/activity-stream#1555]
Currently Bookmarks on the New Tab are seemingly limited to one highlight card in total. This limit needs to be removed, and or modified. Bookmarks that are younger than seventy two hours should automatically be included on the new tab (instantly: without the forty five minute detention period). A minimum of three highlights should be reserved for dynamically selected content. 

This could mean that we only show six ( < seventy two hour old ) bookmark highlights, or allow more than nine total highlights in cases where the user has created more that 6 bookmarks in the last seventy two hours.

**Comment by aaronrbenson** [comment]
Experimenting with bookmarks on New Tab. I added 10 bookmarks over the last hour:
![screen shot 2016-10-13 at 3 02 42 pm](https://cloud.githubusercontent.com/assets/139451/19365111/c336415e-9156-11e6-90dd-368f72692573.png)

After the hour my new tab highlights appear as:
![screen shot 2016-10-13 at 3 03 46 pm](https://cloud.githubusercontent.com/assets/139451/19365122/d815c252-9156-11e6-860d-38ff878e17f2.png)

Seems that I should see more bookmarked items here? 

It's also worth noting that I bookmarked the Atlantic article from the context menu on new tab but it never saved as a bookmark.
